views/overlays/umsk/game16by9.hbs

```diff
@@ -25,6 +25,28 @@
         document.getElementById('rplayer').innerText = data.rplayer || "";
         document.getElementById('lscore').innerText = data.lscore || "";
         document.getElementById('rscore').innerText = data.rscore || "";
+
+        if(data.lCharacter) {
+          var leftIcon = document.createElement('img');
+          leftIcon.id = 'leftCharacter';
+          leftIcon.className = data.lCharacter;
+          leftIcon.style.zoom = '200%';
+          //src is required to remove the dumb border
+          leftIcon.src = 'http://upload.wikimedia.org/wikipedia/commons/c/ce/Transparent.gif';
+          document.getElementById('lplayer').appendChild(leftIcon);
```

Comment:
This was what was happening. https://stackoverflow.com/questions/17851631/jquery-remove-the-innertext-but-preserve-the-html

I'll get a fix soon.